benchmarks/benchmark.py

```diff
@@ -0,0 +1,68 @@
+'''
+numpy module used for vectorization
+scipy for distance calculation
+ribs for generating centroids to be benchmarked
+'''
+import numpy as np
+from scipy.spatial import distance
+
+from ribs.archives import CVTArchive
+
+
+def get_score(centroids, num_samples, seed):
+    '''
+    gets the score for all centroids documented as described in
+    the paper, fast generation of centroids for map elites
+
+    final score equation is different but same results are achieved
+    '''
+    num_centroids = centroids.shape[0]
+    centroid_dim = centroids.shape[1]
+
+    rng = np.random.default_rng(seed=seed)
+    random_samples = rng.random(size=(num_samples, centroid_dim))
+
+    num_closest_pts = np.zeros(num_centroids)
+
+    closest_idx = distance.cdist(random_samples, centroids).argmin(axis=1)
+    assert len(closest_idx) == num_samples
+    for idx in closest_idx:
+        num_closest_pts[idx] += 1
+
+    centroid_vol = num_closest_pts / num_samples
+
+    score = np.sum(np.abs(centroid_vol - 1 / num_centroids))
+
+    return score
+
+
+def main():
+    '''
+    standard main function
+    '''
+    score_seed = 1
+    num_samples = 10000
+    archive = CVTArchive(
```

Comment:
When you implement the other centroid methods, make sure to add them to this file so that it is not only benchmarking CVTArchive centroids